src/QMCHamiltonians/ECPComponentBuilder.2.cpp

```diff
@@ -32,13 +32,13 @@ void ECPComponentBuilder::buildSemiLocalAndLocal(std::vector<xmlNodePtr>& semiPt
   if (grid_global == 0)
   {
     app_error() << "    Global grid needs to be defined." << std::endl;
```

Comment:
app_error() is more for printing form any rank which is unknown.
Since the error is uniform. Please
fuse
```
app_error() << "    Global grid needs to be defined." << std::endl;
```
into `myComm->barrier_and_abort` to print one copy only.

Reply:
fixed